docs/source/literal_types.rst

```diff
@@ -469,6 +469,11 @@ If we forget to handle one of the cases, mypy will generate an error:
 
 Exhaustiveness checking is also supported for match statements (Python 3.10 and later).
```

Comment:
I think this could be expanded instead of adding a new paragraph:

```suggestion
Exhaustiveness checking is also supported for match statements (Python 3.10 and later). With
:option:`--disallow-inexhaustive-match-statements <mypy --disallow-inexhaustive-match-statements>`,
mypy will even warn if match statements are inexhaustive.
```

(not sure that's better than your current approach now that I write it out, actually)